### PR TITLE
updated game section, landing page for PC & mobile , added favicon

### PR DESCRIPTION
**Pull request by kobruhhhhh**

### CODE REVIEW

File: src/components/Maps/Maps.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import styles from "./Maps.module.css";
 import bind from "../../assets/bind.png";
 import haven from "../../assets/haven.png";
 import ascent from "../../assets/ascent.png";
-import split from "../../assets/split.png";
+import split from "../../assets/Split.png";
```
Comment (OverRevvv) [inline src/components/Maps/Maps.jsx:6]: fix the spelling of split use smaller 's' instead of capital 'S'

File: public/index.html [hunk 2]
```diff
@@ -1,22 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
```
Comment (OverRevvv) [inline public/index.html:3]: Reduce changes as much as possible in Index.html use the original code as much as possible to avoid errors.

**Review by OverRevvv — COMMENTED** [review]
### Do the necessary changes and list all the changes that have been done in this PR.

File: public/index.html (hunk 2 above)
Comment (kobruhhhhh) [reply public/index.html:3]: no errors from favicon links or any other extra code snippet....

**Pull request merged**